editor/editor.go

```diff
@@ -35,6 +35,15 @@ func OpenAtLine(number uint) Option {
 	}
 }
 
+// OpenAtEndOfLine opens the file at the end of the line.
+func OpenAtEndOfLine(editor, _ string) (args []string, pathInArgs bool) {
```

Comment:
Shouldn't this return an `Option`?

Reply:
It **is** an option, since there's no arguments, you can simply pass the function

Reply:
oh my bad, i overlooked that part 😅

Reply:
perhaps we could make it explicitly return an `Option`

Reply:
I mean... there's no reason for it as unlike open at line, it doesn't take any options...

there's no great reason against it either, I think